src/api/apiFetch.js

```diff
@@ -60,31 +61,56 @@ export const apiCall = async (
   }
 };
 
-export const apiGet = async (auth, route, params = {}, resFunc, options) =>
+export const apiGet = async (
+  auth: Auth,
+  route: string,
+  params: Object = {},
+  resFunc: (Json) => mixed,
+) =>
   apiCall(auth, `${route}?${encodeAsURI(params)}`, {
     method: 'get',
-  }, resFunc, options);
+  }, resFunc);
```

Comment:
I removed options because it looked like a bug. There is no such thing also I haven't seen any usage of this paramter